palera1n.sh

```diff
@@ -158,11 +159,11 @@ _exit_handler() {
     [ $? -eq 0 ] && exit
     echo "[-] An error occurred"
 
-    cd logs
+    pushd "$rootDir/logs"
```

Comment:
instead, check what the initial wd was and use pushd to easily find the way back later